xml/x509_subjects.xml

```diff
@@ -474,6 +482,14 @@
     <param pos="0" name="service.product" value="vmauthd"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=localhost,OU=Support,O=Palo Alto Networks$">
+    <description>Palo Alto Networks</description>
+    <example>CN=localhost,OU=Support,O=Palo Alto Networks</example>
```

Comment:
Please use the standardized values for this vendor, product, etc. 
You can see an example here - https://github.com/rapid7/recog/blob/fe52bbcc54fdf25c11869c526b75158761de0884/xml/x509_subjects.xml#L1194-L1202

Reply:
We are using PAN-OS currently so this is something to consider.  Would also move this to be with the existing PANOS definition

Reply:
In looking at the CPE data and the Palo Alto website, the value for `os.product` probably should be `PAN-OS`.  Lemme know what works for you and we can bring everything to a consistent state and add CPE mappings if needed.

Reply:
`PAN-OS` is what Nexpose is currently using/expecting for our fingerprinting, so if we think changing recog to `PAN-OS` is correct then that would be perfect for us